### PR TITLE
Dls 5402

### PR DESCRIPTION
**Pull request by SeanMurray1**
[Your pull request description here]

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [ ] Appropriate labels added.
- [ ] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/controllers/NotificationController.scala [hunk 1]
```diff
@@ -172,5 +173,22 @@ class NotificationController @Inject()(private[controllers] val emailConnector:
             }
         }
     }
+
+  def checkReminderType(contactType: Option[ContactType], date: DateTime): (Option[ContactType], String) ={
```
Comment (priyankadhiman3006) [inline app/controllers/NotificationController.scala:177]: I think func name should be changed . It's not just checking the reminder TYpe.  May be something like `getContactTypeAndTemplateVersion`. what do you thnk?

File: test/controllers/NotificationControllerSpec.scala [hunk 2]
```diff
@@ -273,6 +274,60 @@ class NotificationControllerSpec extends PlaySpec with MockitoSugar with ScalaFu
 
     }
 
+    "return correct Content Type" when {
```
Comment (priyankadhiman3006) [inline test/controllers/NotificationControllerSpec.scala:277]: here as well -return correct  contact type and template version

**Review by priyankadhiman3006 — COMMENTED** [review]
Added two comments

File: app/controllers/NotificationController.scala (hunk 1 above)
Comment (SeanMurray1) [reply app/controllers/NotificationController.scala]: Yeah that sounds good will make the change

File: test/controllers/NotificationControllerSpec.scala (hunk 2 above)
Comment (SeanMurray1) [reply test/controllers/NotificationControllerSpec.scala]: yeah I'll make the change.

**Review by priyankadhiman3006 — APPROVED** [review]

**Review by lutnos — APPROVED** [review]
Another time could you please arrange for the copyright year changes to be done separately.  Without that anyone has to do extra work in order to be able to sift the wheat from the chaff

**Review by lutnos — CHANGES_REQUESTED** [review]
Oops, approved the wrong one!

**Comment by lewishay** [comment]
Closing stale PR

**Pull request closed without merging**